lib/pushprocessor.js

```diff
@@ -0,0 +1,226 @@
+var micromatch = require("micromatch");
```

Comment:
How big is the footprint on this (other dependencies that `micromatch` needs?) - `npm list`